mani_skill/utils/scene_builder/replicacad/rearrange/scene_builder.py

```diff
@@ -0,0 +1,274 @@
+"""
+Code for building scenes from the ReplicaCAD dataset https://aihabitat.org/datasets/replica_cad/
+
+This code is also heavily commented to serve as a tutorial for how to build custom scenes from scratch and/or port scenes over from other datasets/simulators
+"""
+
+import json
+import os
+import os.path as osp
+from typing import Dict, List
+from pathlib import Path
+from collections import defaultdict
+import copy
+import itertools
+from functools import cached_property
+
+import numpy as np
+import sapien
+import torch
+import transforms3d
+
+from mani_skill import ASSET_DIR
+from mani_skill.utils.scene_builder.registration import register_scene_builder
+from mani_skill.utils.structs import Actor
+from mani_skill.utils.building import actors
+
+from ..scene_builder import ReplicaCADSceneBuilder
```

Comment:
is this relative import necessary?

Reply:
To clarify: do you mean it should be changed to an absolute import, or if I need to import that class?

Reply:
changed to absolute import